dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -685,6 +686,10 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 					contributionsServiceUrl="https://contributions.guardianapis.com" // TODO: Pass this in
 				/>
 			</Section>
+
+			<Island clientOnly={true} deferUntil="idle">
```

Comment:
Given [this component is rendered in `ArticlePage`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/components/ArticlePage.tsx#L118-L120) for articles, could we have this in [FrontPage component](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/components/FrontPage.tsx)?

Reply:
Thanks for the suggestion! Fixed in c3a3cf8.